src/appl/rangular90_mpi/getinf.f90

```diff
@@ -60,8 +60,9 @@ SUBROUTINE GETINF
             do i = 1,nblock
               write(istde,*) 'Give ICCUT for block',i
     1         READ *, ICCUT(i)
-              IF ((ICCUT(i) <= 1).OR.(ICCUT(i) >= ncfblk(i))) THEN
-                WRITE (istde,*) 'GETINF: ICCUT must be greater than 1',&
+!cychen              IF ((ICCUT(i) <= 1).OR.(ICCUT(i) >= ncfblk(i))) THEN
```

Comment:
I would suggest removing these commented out lines with the old code before merging.